drivers/volume/portworx/portworx.go

```diff
@@ -404,13 +404,15 @@ func (d *portworx) setDriver() error {
 	var endpoint string
 
 	// Try portworx-service first
-	endpoint = d.schedOps.GetServiceEndpoint()
-	if endpoint != "" {
+	endpoint, err = d.schedOps.GetServiceEndpoint()
+	if err == nil {
```

Comment:
nit : add a check for endpoint != "" along with err == nil

Reply:
Done